### ISSUE TITLE
pop-over not appearing or appearing well away from element for Angular 15, Bootstrap 5 and ngx-bootstrap 10.

### ISSUE THREAD

**Issue by rdppjr** [valor-software/ng2-bootstrap#6535]
The only way I can even get the pop-over to appear.  
It will appear only at the bottom of the screen. 
Really partially off the screen
Adding container="body" causes it not to appear at all.

            <button type="button"
              style="border: none"
              popover="Select None (N) to remove."
              popoverTitle="Info"
              placement="auto"
              triggers="mouseenter:mouseleave">
              <i class="fa fa-exclamation-circle" style="color: darkorange"></i>
            </button>

**Comment by RGerhardt-Pressmind** [comment]
@rdppjr The error occurs only with Bootstrap 5.2.x.
If you temporarily switch to Bootstrap 5.1.x, then it works again until the bug in the ngx-bootstrap module is fixed.

**Comment by SvetlanaMuravlova** [comment]
@rdppjr , hi, thank you for your issue, we really haven't checked compatibility with bootstrap 5.2.x, so you please version 5.1.x and soon we will add such enhancement)

**Comment by pgeyman** [comment]
I believe this should be tagged with "Issue" as it is broken functionality. Please fix asap.

**Comment by szymonhel** [comment]
In my case, this style fixed the issue
```
::ng-deep   popover-container {
	position: absolute;
  }
```

**Comment by RGerhardt-Pressmind** [comment]
@pgeyman @szymonhel 

Here is a fix for all

```
.popover {
  position: absolute;
  top: 0;
  left: 0 #{'/* rtl:ignore */'};
}

.popover-arrow {
  position: absolute;
}

.tooltip {
  position: absolute;
}

.tooltip-arrow {
  position: absolute;
}
```

**Comment by julianpoemp** [comment]
same issue with bootstrap v5.1.3. CSS code above fixed it.

**Comment by SvetlanaMuravlova** [comment]
it was fixed in this commit #6543
